pynobo.py

```diff
@@ -585,6 +672,16 @@ def response_handler(self, response):
             dicti = collections.OrderedDict(zip(nobo.API.STRUCT_KEYS_COMPONENT, response[1:]))
             if dicti['zone_id'] == '-1' and dicti['tempsensor_for_zone_id'] != '-1':
                 dicti['zone_id'] = dicti['tempsensor_for_zone_id']
+            serial = dicti['serial']
+            model_id = serial[:3]
+            if model_id in nobo.MODELS:
+                dicti['model'] = nobo.MODELS[model_id]
+            else:
+                dicti['model'] = nobo.Model(
+                    model_id,
+                    "Unknown",
```

Comment:
This syntax does not match any of the Union elements, is it possible to use something like Model.UNKNOWN, or at least "unknown" to match the defined syntaxes?

Reply:
Thanks - this should be `Model.UNKNOWN`.